src/services/theGraph.js

```diff
@@ -20,6 +20,20 @@
 import axios from 'axios';
 import { reportLog } from 'utils/common';
 
+export class GraphQueryError extends Error {
+  subgraphName: string;
+  query: string;
+  responseError: Error;
+
+  constructor(subgraphName: string, query: string, responseError: Error) {
+    // eslint-disable-next-line i18next/no-literal-string
+    super(`The Graph API call to subgraph "${subgraphName}" failed`);
```

Comment:
Just wondering if this would prevent `i18next/no-literal-string`?

```suggestion
    super(`The Graph API call to subgraph \"${subgraphName}\" failed`);
```

Maybe? Not sure if that works...

Reply:
I checked to be sure, but no -- it's the template literal that causes the warning, and it's correct.

Reply:
Super - thanks for clearing that up!